### DISCUSSION TITLE
Windows recognizes my Pi zero as "other device"

### DISCUSSION THREAD

**Discussion by Nexsq** [jayofelony/pwnagotchi#153] [Q&A] [answered]
Yeah so I decided to build myself another pwnagotchi just for fun.
The only problem was that I don't have another sd card so I just decided to take it from my old phone and formatted it, flashed everything was fine to a point where i put it in my pi zero and now windows recognizes it as "BCM2710 Boot" whatever this is -_-
I formatted it as fat32 (I guess it was like this before but just to make sure), flashed it again and still the same, I cant update the drivers because of driver signature enforcement - at least I think that's why as I tried to disable it by restarting my pc with shift pressed then going to troubleshoot, advanced options, startup settings, then pressing F7 but that didn't work, then I tried it again this time pressing just 7 and still windows doesn't see the rndis drivers and just tells me: windows could not find drivers for your device.
Can't select the driver manually, just the path its in so I guess disabling driver signature enforcement did not work, but why??
Can someone help me?

![image_2024-05-09_205600278](https://github.com/jayofelony/pwnagotchi/assets/82839876/73d91cc6-24f2-4f1d-b18c-51d2badf9b57)

![image](https://github.com/jayofelony/pwnagotchi/assets/82839876/d1d030ab-448e-4924-9ae4-6f69b557b818)

**Comment by Nexsq** [comment]
just letting you know I still have this problem

**Reply by rai68** [reply, marked as answer]
https://github.com/jayofelony/pwnagotchi/wiki/Step-2-Connecting#rndis-driver

follow this, if you have the SDcard is not booting, it may be too big
